src/transaction.ts

```diff
@@ -72,27 +74,42 @@ export interface KeyRange {
   endOpen?: Value[];
 }
 
+export interface KeyRang {
```

Comment:
`KeyRange`*

Reply:
Also is this correct? It looks like we actually offer a custom `KeyRange` type and convert each bound to a `IListValue` per https://github.com/googleapis/nodejs-spanner/blob/102cae1970d5e357e3dd7dea6e29b39dd194fcd4/src/transaction.ts#L893

Reply:
removed interface `KeyRang` and used existing one.